.eslintrc.js

```diff
@@ -1,280 +1,214 @@
 module.exports = {
-    'env': {
-        'browser': true,
-        'commonjs': true,
-        'es6': true
+    env: {
+        browser: true,
```

Comment:
Because Prettier did it. It only escapes illegal object key identifiers. I personally like this, but ultimately, I just go with whatever the tool does for me. Had it maintained consistency, I would have deferred to that. I will say this: prettier is a fairly accurate representation of the majority consensus of js style. In other words: you may not like it, but most people do.